packages/common/src/migrations/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Base interface for migratable schemas
+ */
+export interface IMigratableSchema {
```

Comment:
I know these types currently live in `opendata-ui` - does your cleanup over there involve removing the duplication?